Tophat/Utilities/ArtifactDownloader.swift

```diff
@@ -35,6 +37,26 @@ final class ArtifactDownloader: Sendable {
 				try await container.addCopy(of: .rawDownload(fileURL))
 		}
 	}
+
+	private func validateHostTrustIfNeeded(metadata: ArtifactProviderMetadata) async throws {
+		guard metadata.id == "http" else {
+			return
+		}
```

Comment:
A constant for this function?

Reply:
The `http` string.  It's the identifier of the artifact provider that handles HTTP builds in `TophatCoreExtension`.  Technically, it shouldn't be something that Tophat is aware of, but this particular extension may be entitled to special treatment since it's part of Tophat.

Reply:
Ah, I suppose we could add a `ArtifactProviderMetadata.requiresTrustValidation` that gets set when we create the `http` metadata. Though I'm not sure any other ones would need it.